test/crossbrowser.conf.js

```diff
@@ -154,8 +165,16 @@ exports.config = Object.assign(base, {
           platformVersion: '11.0',
           platformName: 'iOS',
           deviceOrientation: 'portrait',
+
+          max_duration: 180,
```

Comment:
These only defined for the iOS case right? What are the default values?

Reply:
When unset crossbrowsertesting.com uses 30 minutes for `max_duration`. The other timeouts here are fed into `wait*` functions which take their default from:

https://github.com/unravelin/ravelinjs/blob/8fb8463e57705604fac76c107ed68a8d0ad813b3/test/base.conf.js#L91-L92